EIPS/eip-6789.md

```diff
@@ -0,0 +1,88 @@
+---
+eip: 6789
+title: Rename `gas` to `mana`
+description: This EIP picks up on Vitalik Buterin's (@vbuterin) original proposal in 2015 to rename `gas` to `mana`
+author: Pascal Caversaccio (@pcaversaccio)
+discussions-to: https://ethereum-magicians.org/t/eip-6789-rename-gas-to-mana/13570
+status: Draft
+type: Standards Track
+category: Interface
+created: 2023-03-27
+---
+
+## Abstract
+
+This EIP picks up on Vitalik Buterin's (@vbuterin) original proposal in 2015 to rename `gas` to `mana`.
+
+## Motivation
+
+Vitalik's original motivation is threefold and is still valid today:
+
+- `mana` more accurately reflects the increased environmental friendliness of Proof-of-Stake (PoS),
+- `mana` is generally understood to be ephemeral and non-transferable, and thus represents accurately the characteristic of how `gas` is actually used within Ethereum, and
+- `mana` is generally considered renewable, whilst (natural) gas is considered non-renewable.
```

Comment:
```suggestion
- `mana` reflects the increased environmental friendliness of Proof-of-Stake;
- `mana` is generally understood to be ephemeral and non-transferable, which better represents the concept of `gas`; and
- `mana` is generally portrayed as renewable, while (natural) `gas` is non-renewable.
```

Reply:
FWIW, I abbreviated PoS since I re-used it below in the rationale, but since am gonna switch both sections anyways I will use your suggestion.